client/controller/index.node.js

```diff
@@ -47,3 +47,11 @@ const ProviderWrappedLoggedOutLayout = ( {
  * `context.primary` and `context.secondary` to populate it.
  */
 export const makeLayout = makeLayoutMiddleware( ProviderWrappedLoggedOutLayout );
+
+/**
```

Comment:
Webpack will replace `client/controller/index.node.js` (the file used by VSCode and ESLint) with `client/controller/index.web.js`. However, they don't have the same API, as the web version exposes many functions that the Node version doesn't. Those "extra" functions are used everywhere in our repo.

Adding a noop implementation of those functions in `client/controller/index.node.js` is the easiest solution.

An alternative would be to add a `//eslint-disable-next-line import/named` in every file where those functions are used, but that's a huge and noisy change. Happy to do it though, if we decide that's a better solution.

Reply:
This solution makes sense to me :+1: